app/import_data_v2.py

```diff
@@ -70,12 +73,12 @@ def insert_meta(input, type, company_id):
                 company_size = '1-10'
 
             # Check if the city is already in the cities dict
-            if agency['city'].capitalize() not in cities.values():
+            if agency['city'].title() not in cities.values():
```

Comment:
In the endpoint you use `.capitalize()` here you use `.title()` 🤔

Isn't that going to create issues when searching?